docs/index.md

```diff
@@ -25,15 +29,15 @@ ARGUMENTS
      -l, --log-file  <arg>     The name of the logfile in csv format. If not provided a file
                                easy-fedora2vault-<timestamp>.csv will be created in the home-dir of the user.
                                (default = /home/vagrant/easy-fedora2vault-2020-02-02T20:20:02.000Z.csv)
-     -o, --output-dir  <arg>   Empty directory in which to stage the created AIP bags. It will be created if it
+     -o, --output-dir  <arg>   Empty directory in which to stage the created IPs. It will be created if it
                                doesn't exist.
      -s, --strict              If provided, the transformation will check whether the datasets adhere to the
                                requirements of the chosen transformation.
      -h, --help                Show help message
      -v, --version             Show version of this program
 
     trailing arguments:
-      transformation (required)   The type of transformation used. Possible values: simple, thematische-collectie.
+     transformation (required)   The type of transformation used: simple-AIP, simple-SIP, thematische-collectie.
```

Comment:
Would collections also need the AIP/SIP variants?

Reply:
we decided to make the output-type (AIP/SIP) a separate parameter.